nanodbc/nanodbc.cpp

```diff
@@ -2936,7 +2968,8 @@ inline void result::result_impl::get_ref_impl<string>(short column, string& resu
             // Type is unicode in the database, convert if necessary
             const SQLWCHAR* s =
                 reinterpret_cast<SQLWCHAR*>(col.pdata_ + rowset_position_ * col.clen_);
-            const string::size_type str_size = col.cbdata_[static_cast<size_t>(rowset_position_)] / sizeof(SQLWCHAR);
+            const string::size_type str_size =
```

Comment:
Unless you did run clang-format, this looks like an accidental change and should not be mixed in.

Reply:
Yes I've run clang-format. Should i revert these changes?

Reply:
Oh, I see. Then possibly I it's my bad and I had forgotten to run clang-format before a commit.

>  Should i revert these changes?

Best would be if style changes are submitted as separate commits/PRs.

If it would not be a problem for you to revert the style changes, then I could run clang-format, commit, then you could rebase your `ChrisBFX:string_refactor` to update this PR.

If reverting the style changes is a trouble, we could swallow it this time, unless @lexicalunit objects.

Reply:
I'm ok with style changes being applied with PRs that add functionality sometimes. It's not ideal but it's not the end of the world. We shouldn't make it a habit if we can avoid it easily.